source/usage-and-configuration/swagger/introduction.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Swagger
+---
+
+# Swagger
+
+The theme provides tools to read and parse swagger files and generate documentations out of it.
+
+## Tags
+It provides following two tags which can grab the swagger schema from local or a remote location and create beautiful and easy to read documents.
```

Comment:
It provides the following two tags. . . schema from a local or remote location. . .

Reply:
👍